lib/CMakeLists.txt

```diff
@@ -46,7 +46,8 @@ add_library(gnuradio-adapt SHARED ${adapt_sources})
 target_link_libraries(gnuradio-adapt gnuradio::gnuradio-runtime
   gnuradio::gnuradio-filter
   gnuradio::gnuradio-blocks
-  gnuradio::gnuradio-fft)
+  gnuradio::gnuradio-fft
+  VOLK_LIBRARIES)
```

Comment:
This should be evaluated as a variable `${VOLK_LIBRARIES}`.

Reply:
sorry, my fault ;-)
fixed